build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 
 group = 'uk.ac.ebi.ena.sra'
 archivesBaseName = 'sra-xml'
-version = '1.9.0'
+version = '1.9.1'
```

Comment:
- The gradle build version is 1.9.1 but the version in XDSs is 1.9.0. Perhaps we don't need a version in the XSDs at all.